docs/explanation/archive.rst

```diff
@@ -37,7 +37,7 @@ Series
 ------
 
 A **series** refers to the packages that target a specific Ubuntu version. A
-series is usually referred to by its :term:`codename <Codename>`.
+series is usually referred to by its :term:`code name <Codename>`.
```

Comment:
Issue: I saw a change  "codename" -> "code name" in another file. This has to be consistent.
```suggestion
series is usually referred to by its :term:`code name <code name>`.
```

TODO: The glossary entry has to be changed too.